### ISSUE TITLE
Need some tips for how the season hash values update in the profile call

### ISSUE THREAD

**Issue by YourFavoritePerson** [Bungie-net/api#1317]
For most people, we've been telling them to log into the game after getting BL, then logging out completely.  It's been updating their profile call's season hash list when they do that, but some people are saying they've done that and it still isn't updating.

Fish Jesus here said he logged out completely but the API isn't showing that he owns BL.  Is there any other way to force that hash array to update?

```
{{
  "Response": {
    "profile": {
      "data": {
        "userInfo": {
          "crossSaveOverride": 0,
          "applicableMembershipTypes": [
            3
          ],
          "isPublic": true,
          "membershipType": 3,
          "membershipId": "4611686018483189742",
          "displayName": "Fish Jesus"
        },
        "dateLastPlayed": "2020-11-12T01:16:08Z",
        "versionsOwned": 63,
        "characterIds": [
          "2305843009402776072",
          "2305843009569746482"
        ],
        "seasonHashes": [
          1743682818
        ],
        "currentSeasonHash": 2809059427,
        "currentSeasonRewardPowerCap": 1260
      },
      "privacy": 1
    },
```

This guy too (SpartanFTW):
```
{{
  "Response": {
    "profile": {
      "data": {
        "userInfo": {
          "crossSaveOverride": 0,
          "applicableMembershipTypes": [
            3
          ],
          "isPublic": true,
          "membershipType": 3,
          "membershipId": "4611686018482735803",
          "displayName": "SpartanFTW"
        },
        "dateLastPlayed": "2020-11-12T03:21:53Z",
        "versionsOwned": 31,
        "characterIds": [
          "2305843009399845527",
          "2305843009399845556"
        ],
        "seasonHashes": [],
        "currentSeasonHash": 2809059427,
        "currentSeasonRewardPowerCap": 1260
      },
      "privacy": 1
    },
```

**Comment by jshaffstall-bng** [comment]
Now that some more time has passed, are you still hitting this issue?

**Comment by YourFavoritePerson** [comment]
Yes.  Spartan DM'ed me this morning saying he tried again and it still wasnt working.  I'll follow up with him to make sure he went to orbit and closed the game before attempting to rejoin the clan.  Someone said the same thing a couple hours ago, but I didnt go through the normal troubleshooting steps with them yet.

I'll post back when I get responses from them

**Comment by YourFavoritePerson** [comment]
The new guy from 11am said he tried the normal steps and it didnt work.  he's VadaVada - 
```
 "Response": {
    "profile": {
      "data": {
        "userInfo": {
          "crossSaveOverride": 0,
          "applicableMembershipTypes": [
            3
          ],
          "isPublic": true,
          "membershipType": 3,
          "membershipId": "4611686018507588836",
          "displayName": "VadaVada"
        },
        "dateLastPlayed": "2020-11-12T21:17:27Z",
        "versionsOwned": 63,
        "characterIds": [
          "2305843009695734788"
        ],
        "seasonHashes": [
          2809059424
        ],
        "currentSeasonHash": 2809059427,
        "currentSeasonRewardPowerCap": 1260
      },
      "privacy": 1
    },```

**Comment by YourFavoritePerson** [comment]
Spartan just tried going to orbit and quitting the game. his hash's still didnt update.

**Comment by YourFavoritePerson** [comment]
We're now up to 7 people who potentially have this issue. 2 people have had their season hashes update to what they should be so they aren't on this list:

formatted as [membershipType]-[membershipId]
3-4611686018509185262
2-4611686018452701842
3-4611686018482735803
3-4611686018483152395
3-4611686018487060630
3-4611686018487676664
3-4611686018507483800

**Comment by kramrm** [comment]
Don't forget that `seasonHashes` are which seasons are purchased (Hunt, Arrival, Worthy, etc).  These stay with the profile and the value is separate from which DLCs have been purchased.  The `versionsOwned` field should indicate which DLCs are owned (Forsaken, Shadowkeep, etc) and are locked to the platform purchased. This value can change if a cross save user logs into a platform where they own a different set of DLCs.  However the documentation doesn't appear to have been updated to include a value for Beyond Light.

https://bungie-net.github.io/multi/schema_Destiny-DestinyGameVersions.html#schema_Destiny-DestinyGameVersions

**Comment by YourFavoritePerson** [comment]
We were having them log in and out of the game before we put overrides in for them.  It shouldn't be caused by them logging in on another platform.

The versionsOwned variable didnt update for beyond light.  it still maxes out at the max Y3 value.

**Comment by kramrm** [comment]
I do agree that there doesn't seem to be a flag for Beyond Light in the DestinyGameVersions enum.  Not sure if @jshaffstall-bng has any insight on this.

I had logged in last night on my Stadia profile, which doesn't have any DLC purchased.  At that time my profile showed versionsOwned=7, which is what I'd expect for that.  Once I logged into my Steam profile and visited a destination, my profile updated to show versionsOwned=63, verifying that I've got all DLC up through Shadowkeep.
